infrastructure/variables.tf

```
@@ -15,10 +15,14 @@ variable "env" {
   type = "string"
 }
 
-variable "ilbIp" {}
-
 variable "subscription" {}
 
 variable "common_tags" {
   type = "map"
 }
+
+variable "appinsights_location" {
+  type        = "string"
+  default     = "West Europe"
```

probably should just use the location variable, (app insights didn't use to be in uk south, but since we're recreating it this is the right time to fix it)

```suggestion
  default     = "UK South"
```